File: backjoon/stage02/5543_상근날드.go
package main

import (
	"fmt"
)

func main() {
	burgers := make([]int, 3)
	for i := range burgers {
		fmt.Scanf("%d", &burgers[i])
	}

	beverages := make([]int, 2)
	for i := range beverages {
		fmt.Scanf("%d", &beverages[i])
	}

	var minBurger, minBeverage = 2000, 2000
	for _, burger := range burgers {
		if burger < minBurger {
			minBurger = burger
		}
	}
	for _, beverage := range beverages {
		if beverage < minBeverage {
			minBeverage = beverage
		}
	}
	fmt.Println(minBurger + minBeverage - 50)
}

File: baekjoon/stage05/5543_상근날드.py
import sys

min_burger, min_beverage = 2000, 2000

for i in range(3):
    price = int(sys.stdin.readline())
    if price < min_burger:
        min_burger = price

for i in range(2):
    price = int(sys.stdin.readline())
    if price < min_beverage:
        min_beverage = price
        
print(min_burger + min_beverage - 50)